rust/gui-client/src-tauri/src/client/logging.rs

```diff
@@ -54,6 +54,7 @@ pub(crate) fn setup(log_filter: &str) -> Result<Handles, Error> {
         );
     }
     LogTracer::init()?;
+    tracing::debug!(?log_path, "Log path");
```

Comment:
Used to debug CI. It doesn't hurt to leave it in, since switching between sudo and non-sudo makes the logs show up in different places, and it's easier just to double-check in stderr on a dev system with a terminal.